modules/stochastic_tools/doc/content/source/surrogates/PODReducedBasisTrainer.md

```diff
@@ -0,0 +1,307 @@
+# PODReducedBasisTrainer
+
+!syntax description /Trainers/PODReducedBasisTrainer
+
+## Overview
```

Comment:
I would recommend citing a POD paper somewhere for reference, maybe this one would be good:
```
@Inbook{pinnau2008model,
author="Pinnau, Ren{\'e}",
title="Model Reduction via Proper Orthogonal Decomposition",
bookTitle="Model Order Reduction: Theory, Research Aspects and Applications",
year="2008",
publisher="Springer Berlin Heidelberg",
address="Berlin, Heidelberg",
pages="95--109",
chapter="5",
abstract="In many fields of science and engineering, like fluid or structural mechanics and electric circuit design, large--scale dynamical systems need to be simulated, optimized or controlled. They are often given by discretizations of systems of nonlinear partial differential equations yielding high--dimensional discrete phase spaces. for this reason during the last decades research was mainly focused on the development of sophisticated analytical and numerical tools to understand the overall system behavior. Not surprisingly, the number of degrees of freedom for simulations keptpace with the increasing computing power. But when it comes to optimal design or control the problems are in general to large to be tackled with standard techniques. Hence, there is a strong need for model reduction techniques to reduce the computational costs and storage requirements. They should yield low--dimensional approximations for the full high--dimensional dynamical system, which reproduce the characteristic dynamics of the system.",
isbn="978-3-540-78841-6",
doi="10.1007/978-3-540-78841-6_5",
}
```